docs/src/operate/lifecycle/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Lifecycle
```

Comment:
I'm not convinced with the name, `Entity Registration` might be better, and it would align with the reference doc usage, http://localhost:3000/thin-edge.io/next/references/mqtt-api/#entity-registration.

Reply:
And this *deregistration* section is a call for a new *registration* page.

Reply:
Yeah, currently most of it is in the references doc, which definitely isn't the first thing an end-user would look for.